cw_bitcoin/lib/litecoin_wallet.dart

```diff
@@ -118,4 +127,69 @@ abstract class LitecoinWalletBase extends ElectrumWallet with Store {
 
     return 0;
   }
+
+  @override
+  Future<String> signMessage(String message, {String? address = null}) async {
+    final index = address != null
+        ? walletAddresses.allAddresses.firstWhere((element) => element.address == address).index
+        : null;
+    final HD = index == null ? hd : hd.derive(index);
+    final priv = ECPrivate.fromHex(HD.privKey!);
+    String messagePrefix = '\x19Litecoin Signed Message:\n';
+    return priv.signMessage(utf8.encode(message), messagePrefix: messagePrefix);
+  }
```

Comment:
It seems like the library you are using cannot sign Litecoin Messages. I get the error `invalid message prefix. message prefix should start with 0x18`, but Litecoins messagePrefix starts with `\x19`